### PR TITLE
Function Access Change

### PR DESCRIPTION
**Pull request by SamRagusa**
- Made core cloud data functions accessible from `endaq.cloud.*` rather than `endaq.cloud.core.*`

### CODE REVIEW

**Comment by SamRagusa** [comment]
@CrepeGoat  is this going to mess up the docs?  Also will we run into the same problem we had with `endaq.plot` where we had to define `__all__`?

**Review by StokesMIDE — CHANGES_REQUESTED** [review]
Add a newline to the end of the file and I'd say it's good to go.

**Comment by CrepeGoat** [comment]
> is this going to mess up the docs?

We'd have to edit them to not look for the `core` module, yeah. but tbh if we're going to make changes, I'd rather do it sooner before people get comfortable with having a certain module in a certain spot.

> Also will we run into the same problem we had with `endaq.plot` where we had to define `__all__`?

Probably, yeah. Even if we don't explicitly see the problem come up, I'd like to do the `__all__` bit anyway? unless we find a better way to reject unwanted imports... (but I will note it's in a few files in `endaq.ide` as well, so it's not a new/strange thing per se.)

File: endaq/cloud/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from .core import *
```
Comment (CrepeGoat) [inline endaq/cloud/__init__.py]:
```suggestion
from .core import *

```

**Review by CrepeGoat — CHANGES_REQUESTED** [review]
suggested thing, and also adding `__add__` pre-emptively to "core.py"

**Comment by StokesMIDE** [comment]
+1 on the addition of `__all__`. I think it's good practice for a public package like this.

**Comment by SamRagusa** [comment]
I've set the `__all__` variable on endaq.cloud.core

**Review by CrepeGoat — APPROVED** [review]
looks good

File: endaq/cloud/core.py [hunk 2]
```diff
@@ -14,18 +14,16 @@
...
-# ==============================================================================
+__all__ = [
+    'EndaqCloud',
+    'count_tags',
+    'json_table_to_df',
+]
 
 ENV_PRODUCTION = "https://qvthkmtukh.execute-api.us-west-2.amazonaws.com/master"
 ENV_STAGING = "https://p377cock71.execute-api.us-west-2.amazonaws.com/staging"
 ENV_DEVELOP = "https://mnsz98xs64.execute-api.us-west-2.amazonaws.com/develop"
```
Comment (StokesMIDE) [inline endaq/cloud/core.py:25]: These really shouldn't be in the distributed package

**Review by StokesMIDE — APPROVED** [review]

File: endaq/cloud/core.py (hunk 2 above)
Comment (SamRagusa) [reply endaq/cloud/core.py:25]: Why, they can't access anything from within them.   What are you worried could happen if they are?   Having these be easily accessible to the internal Mide people is helpful

**Pull request merged**